ts/src/gemini.ts

```diff
@@ -1655,10 +1654,7 @@ export default class gemini extends Exchange {
     }
 
     nonce () {
-        const nonceMethod = this.safeString (this.options, 'nonce', 'milliseconds');
-        if (nonceMethod === 'milliseconds') {
```

Comment:
@ttodua we should not use seconds by default IMO, what if you want to place 2 orders within the same second?

Reply:
and having a way of configuring it makes it more flexible

Reply:
and have you tried making a request with milliseconds? for me, it didnt work, so I couldnt understand what's point of having milliseconds, as long as exchange doesnt support. but again, as I dont have direct access to gemini API, i might be missing something, but if that's true what i say, then there is no point having configurable option if it doesnt work.

Reply:
I have added milliseconds as fraction after dot, and seems to work, so you can similarly fire several orders in same second

Reply:
@ttodua The problem is this will be a breaking change for our users, if they were using milliseconds, and now out of nowhere we send seconds the API will reject every single requests because seconds << milliseconds

Reply:
using ms works fine on my end:
```
 n gemini fetchMyTrades "LTC/USD" --sandbox
2024-04-03T10:43:23.361Z
Node.js: v18.18.0
CCXT v4.2.87
gemini.fetchMyTrades (LTC/USD)
2024-04-03T10:43:25.076Z iteration 0 passed in 667 ms

[]
0 objects
2024-04-03T10:43:25.076Z iteration 1 passed in 667 ms
```

Reply:
Will restore this part, you're blocking the precisions issue with this, which is not worth it (the user reporting it was not facing timestamp issues for sure)